kytos/core/auth.py

```diff
@@ -168,200 +244,83 @@ def _authenticate_user(self):
             result = f"Incorrect username or password: {exc}"
             return result, HTTPStatus.UNAUTHORIZED.value
 
-    def _find_user(self, uid):
-        """Find a specific user using Storehouse."""
-        response = {}
-
-        def _find_user_callback(_event, box, error):
-            nonlocal response
-            if not box:
-                response = {
-                    "answer": f'User with uid {uid} not found',
-                    "code": HTTPStatus.NOT_FOUND.value
-                }
-            elif error:
-                response = {
-                    "answer": "User data cannot be shown",
-                    "code": HTTPStatus.INTERNAL_SERVER_ERROR.value,
-                }
-            else:
-                response = {
-                    "answer": {"data": box.data},
-                    "code": HTTPStatus.OK.value,
-                }
-
-        content = {
-            "box_id": uid,
-            "namespace": self.namespace,
-            "callback": _find_user_callback,
-        }
-        event = KytosEvent(name="kytos.storehouse.retrieve", content=content)
-        self.controller.buffers.app.put(event)
-        while True:
-            time.sleep(0.1)
-            if response:
-                break
-        return response["answer"], response["code"]
+    def _find_user(self, username):
+        """Find a specific user using MongoDB."""
+        user = self.user_controller.get_user(username)
+        if user == {}:
```

Comment:
```suggestion
        if not user:
```

if not user is more idiomatic here, I think.

Reply:
Yep, they have the same result.